src/app/shared/components/menu/main-menu/main-menu.component.ts

```
@@ -105,7 +107,27 @@
+          } else if (integrationName === InAppIntegration.XERO) {
+            accountingIntegrationApp = AccountingIntegrationApp.XERO;
+          } else {
+            return;
+          }
+
+          const payload = {
+            callbackUrl: integrationCallbackUrlMap[accountingIntegrationApp][0],
+            clientId: integrationCallbackUrlMap[accountingIntegrationApp][1]
+          };
+
+          console.log({payload})
```

ESLint

[error] Unexpected console statement.

(no-console)